react-native.config.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+const deactivatedAutolinkingModules = [
```

Comment:
Ah okay, that explains previous occurrence in `MainApplication.java`.

Reply:
Could you remind me the reason why this is necessary?

Reply:
If I don't do this, I get an error telling me to unlink the native modules, when I try to unlink the native modules, It says the module is not linked. When I remove it from build, and wait for it to be autolinked, it is not.